c10/util/win32-headers.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN
+#endif
+#ifndef NOMINMAX
+#define NOMINMAX
+#endif
+#ifndef NOKERNEL
+#define NOKERNEL
+#endif
+#ifndef NOUSER
+#define NOUSER
+#endif
+#ifndef NOSERVICE
+#define NOSERVICE
+#endif
+#ifndef NOSOUND
+#define NOSOUND
+#endif
+#ifndef NOMCX
+#define NOMCX
+#endif
+#ifndef NOGDI
+#define NOGDI
+#endif
+
+#include <windows.h>
+#include <dbghelp.h>
+
+#undef DELETE

```

Comment:
`VOID` is used in `c10/util/Backtrace.cpp`. Personally I prefer `void` over `VOID`. If no `VOID` is used, it can be undef-ed.

Reply:
Please feel free to do that.

Reply:
There's no good reason for `VOID` to be used, so yeah, kill it and undef it.